src/configs/config-variables.ts

```diff
@@ -21,6 +21,15 @@ type ConfigVariables = {
   connectTestAccountPassword: string;
   serviceName: string;
   lightstepAccessToken: string;
+  fwlTracingCollectors: {
+    type: string;
+    serviceName: string;
+    authorizationHeader?: {
+      key: string;
+      value: string;
+    };
+    url: string;
+  }[];
```

Comment:
I would have prefer to do
```suggestion
  fwlTracingCollectors: TracingConfig["collectors"];
```
but since it can be `undefined`, it messes up with the instanciation of the tracer